### DISCUSSION TITLE
Getting a '402 Payment Required' error when publishing private packages

### DISCUSSION THREAD

**Discussion by petecollctiv** [community/community#68196] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We've just started getting this '402 Payment  error when publishing. And yes we have paid our bill. 

Our Organization account is weirdly saying we're on the 'Free' plan despite paying for a Teams subscription.

Any ideas?

**Comment by petecollctiv** [comment]
closing

**Reply by augustoarnold13** [reply]
what is the issue? I`m having the same today

**Reply by petecollctiv** [reply]
Looks like a degradation of the service. The issue is fixed for us now. https://status.npmjs.org/incidents/95vmqpd2b66m

**Reply by augustoarnold13** [reply]
still happening to us, will check with them, Thanks!

**Reply by MichaelBischoffArvato** [reply]
Happened to me as well today

**Reply by kumaarn** [reply]
any solution for fix? currently its happening to us

**Reply by fhoess** [reply]
you must set the budget on Organization level: (even, if you do not need that budget)
![image](https://github.com/user-attachments/assets/9280fb79-f79d-4d41-acba-4b748a0ccf47)

**Comment by fhoess** [comment]
see my solution with the budget
